app/views/self_service/firms/questionnaire/_languages.html.erb

```diff
@@ -0,0 +1,32 @@
+<fieldset class="form__group" data-dough-component="LanguageSelector">
+  <legend class="l-questionnaire__legend"><%= t('self_service.firm_form.languages_heading') %></legend>
+
+  <div class="is-hidden language-selector__select" data-dough-language-selector-template>
+    <%= firm_language_select nil, class: 't-languages' %>
+    <%= firm_language_delete_button %>
+  </div>
+
+  <%= f.form_row :other_services do %>
+    <%= f.errors_for :other_services %>
+
+    <div class="language-selector__selectors" data-dough-language-selector-selectors>
+      <% if f.object.languages.empty? %>
+        <div class="language-selector__select" data-dough-language-selector-selector>
+          <%= firm_language_select nil, class: 't-languages' %>
+          <%= firm_language_delete_button %>
+        </div>
+      <% end %>
+
+      <% f.object.languages.each do |language| %>
+        <div class="language-selector__select" data-dough-language-selector-selector>
+          <%= firm_language_select language, class: 't-languages' %>
+          <%= firm_language_delete_button %>
+        </div>
+      <% end %>
+    </div>
+
+    <button type="button" class="button-link language-selector__add" data-dough-language-selector-add-language>
```

Comment:
Good point — I've added some CSS and such to make the whole thing behave better without JS. It just shows a single select box now, no non-functional buttons.